org.adempiere.base/src/org/compiere/db/AdempiereDatabase.java

```diff
@@ -488,6 +488,13 @@ public default String getSQLCreate(MTable table)
 		return sb.toString();
 	}	//	getSQLCreate
 	
+	/**
+	 * Get SQL Statement to store Blob 

```

Comment:
Replace this with "Convert blob to hex encoded string and return SQL function that will convert the hex encoded string back to blob"